test/spoke/integration/SyncDeposit.t.sol

```diff
@@ -176,21 +176,10 @@ contract SyncDepositTest is SyncDepositTestHelper {
         );
 
         if (snap) {
-            vm.startSnapshotGas("SyncDepositVault", "deposit_withoutQueue");
+            vm.startSnapshotGas("SyncDepositVault", "deposit");
         }
         // _assertDepositEvents(syncVault, shares.toUint128(), pricePoolPerShare, pricePoolPerAsset);
-        syncVault.deposit(amount / 2, self);
-        if (snap) {
-            vm.stopSnapshotGas();
-        }
-
-        balanceSheet.setQueue(syncVault.poolId(), syncVault.scId(), true);
```

Comment:
IIUC, the measure of this was wrong. We're already queuing by default, so the previous measurement was also taking into account that it was queuing